src/ResourceManagement/Sql/SqlManagement/Generated/Models/ExportRequestParameters.cs

```diff
@@ -0,0 +1,40 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System;
+using System.Linq;
+using Microsoft.Azure.Management.Sql.Models;
+
+namespace Microsoft.Azure.Management.Sql.Models
+{
+    /// <summary>
+    /// Export Azure SQL Database parameters.
+    /// </summary>
+    public partial class ExportRequestParameters : ImportExportRequestParametersBase
+    {
+        /// <summary>
+        /// Initializes a new instance of the ExportRequestParameters class.
+        /// </summary>
+        public ExportRequestParameters()
```

Comment:
@llali What is the usage of a class that does not have any property or method but just one empty public constructor?